crates/lib/src/identity.rs

```diff
@@ -63,36 +71,32 @@ impl Identity {
 
     #[doc(hidden)]
     pub fn __dummy() -> Self {
```

Comment:
I know this wasn't added in this PR but I looked at usages, and I'm unclear why we even need those functions instead of just using `::ZERO`.